examples/k8s/value_tso_HA.yaml

```diff
@@ -0,0 +1,449 @@
+    storage:
+      localDisk:
+        pvcSpec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 50Gi
+          storageClassName: openebs-hostpath #replace to your storageClassName
+      log:
+        pvcSpec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 10Gi
+          storageClassName: openebs-hostpath #replace to your storageClassName
+    configOverwrite:
+      logger:
+        level: trace
+      disk_cache_strategies:
+        simple:
+          lru_max_size: 429496729600 # 400Gi
+      # timezone: Etc/UTC
+    hdfs3Config:
+      dfs.client.metrics.enable: true
+      dfs.client.metrics.use_domain_socket: true
+      dfs.default.blocksize: "268435456"
+      dfs.read.hedge_read.interval_ms: 120
+      input.connect.timeout: 10000
+      input.read.max.retry: 3
+      input.read.timeout: 6000
+      input.write.timeout: 10000
+      output.close.timeout: 256000
+      output.connect.timeout: 10000
+      output.default.write.retry: 3
+      output.read.timeout: 120000
+      output.write.timeout: 256000
+      rpc.client.connect.retry: 3
+      rpc.client.connect.timeout: 10000
+      rpc.client.read.timeout: 20000
+      rpc.client.timeout: 20000
+      rpc.client.write.timeout: 20000
+
+  virtualWarehouses:
+    - <<: *defaultWorker
+      name: vw_default
+      replicas: 1
+    - <<: *defaultWorker
+      name: vw_write
+      replicas: 1
+
+  commonEnvs:
+    - name: MY_POD_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: "metadata.namespace"
+    - name: MY_POD_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: "metadata.name"
+    - name: MY_UID
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: "metadata.uid"
+    - name: MY_POD_IP
+      valueFrom:
+        fieldRef:
+          fieldPath: "status.podIP"
+    - name: MY_HOST_IP
+      valueFrom:
+        fieldRef:
+          # fieldPath: "status.hostIP"
+          fieldPath: "status.podIP"
+    - name: CONSUL_HTTP_HOST
+      valueFrom:
+        fieldRef:
+          fieldPath: "status.hostIP"
+
+  additionalEnvs: [ ]
+
+  additionalVolumes:
+    volumes: [ ]
+    volumeMounts: [ ]
+
+  postStart: ""
+  preStop: ""
+  readiness: ""
+  liveness: ""
+
+  ingress:
+    enabled: false
+    className: ""
+    annotations: { }
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: byconity-example.local
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+    tls: [ ]
+    #  - secretName: byconity-example-tls
+    #    hosts:
+    #      - byconity-example.local
+
+# For more detailed usage, please check fdb-kubernetes-operator API doc: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/docs/cluster_spec.md
+fdb:
+  enabled: true
+  enableCliPod: true
+  version: 7.1.15
```

Comment:
let change it to 7.1.27

Reply:
should i need to test it before release? i'm still using 7.1.15